packages/integration-sdk/CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to
 
 ## Unreleased
 
+### Changed
+
+- Stop handling `IntegrationDuplicateKeyError` as fatal
```

Comment:
```suggestion
- Stop handling `IntegrationDuplicateKeyError` as fatal. Previously, this type of error would terminate the integration completely and no data would be ingested. Now, the step that raises this error will not complete, and any steps that depend on the failed step will not execute, but all other steps will complete and partial datasets will be processed.
```

Reply:
I think in the future it could be worthwhile to have the `jobState` just ignore the duplicate data and have it send us something in Sentry, so that it can ingest all the data except the duplicate, and we have what we need in Sentry to investigate the duplicate and adjust the integration to stop producing them. That said, this solution is a great step forward to avoid a total crash!